Bearded.Utilities/Geometry/Angle.cs

```diff
@@ -358,6 +358,9 @@ public override int GetHashCode()
         public string ToString(string? format, IFormatProvider? formatProvider)
             => $"{Degrees.ToString(format, formatProvider)}°";
 
+        public string ToString(string? format)
+            => $"{Degrees.ToString(format)}°";
```

Comment:
This applies for all these new methods: Since this uses the exact same format string as the version with IFormatProvider, it would be great if the new method would call that other one instead, either passing in null or CultureInfo.CurrentCulture (I'm not sure which is preferred, but I think they do basically the same).

Reply:
I was actually thinking to do it that way, but for some mystical reason I went with this one, I'll get right on it.